crates/install-wheel-rs/src/wheel.rs

```diff
@@ -216,20 +212,33 @@ pub(crate) fn write_script_entrypoints(
                 .unwrap_or(&entrypoint.name)
                 .to_string()
                 + ".exe";
-            bin_rel().join(script_name)
+
+            scripts.join(script_name)
         } else {
-            bin_rel().join(&entrypoint.name)
+            scripts.join(&entrypoint.name)
         };
 
+        let entrypoint_relative = pathdiff::diff_paths(&entrypoint_absolute, site_packages)
```

Comment:
How's that different from `Path::strip_prefix`?

If we need it, can we copy the function into `uv_fs` (https://github.com/Manishearth/pathdiff/blob/a78acde0955a556b2bdf11e6a9efae459c4bcf80/src/lib.rs#L34-L77)? The repo looks abandoned and there are no docs.

Reply:
Trying to understand, why would `strip_prefix` work here? The entrypoint is not a subdirectory of `site_packages` (if you look at `bin_rel()` previously, it started with `../..`).

Reply:
I see, makes sense